ruby/vampires.rb

```diff
@@ -0,0 +1,52 @@
+puts "How many employees are you processing?"
+num_employees = gets.chomp.to_i
+
+num_employees.times do
+		
+	puts "What is your name?"
+	employee = gets.chomp
+	puts "How old are you?"
+	age = gets.chomp.to_i
+	puts "What year were you born?"
+	birthyear = gets.chomp.to_i
+		if 2017 - age == birthyear
+			agecorrect = true
+		else
+			agecorrect = false
+		end
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
+	garlic = gets.chomp
+		if garlic == "y"
+			garlic = true
+		else garlic = false
+		end
+	puts "Would you like to enroll in the company's health insurance?"
+	insurance = gets.chomp
+		if insurance == "y"
+			insurance = true
+		else insurance = false
+		end
+	allergy = nil
```

Comment:
It's a little funky to declare a variable outside of the loop, so look into restructuring the format to have the puts outside of the loop.